.github/workflows/neard_custom_release.yml

```diff
@@ -44,7 +44,9 @@ jobs:
           fetch-depth: 0
 
       - name: Neard binary build and upload to S3
-        run: ./scripts/binary_release.sh
+        env:
+          release_type: ${{ github.event.inputs.release }}
```

Comment:
this is defined at line 8

Reply:
why go through the environment variable?

Reply:
Yeah it was my practice for other reasons but it can directly be used in run command line. Changed.